share/autostart

```diff
@@ -6,6 +6,8 @@ hc() {
     herbstclient "$@"
 }
 
+if [ -z $TERMINAL ]; then export TERMINAL="xterm"; fi
```

Comment:
Exporting a variable is a side effect on any commands called from autostart. I understand you want autostart to use your $TERMINAL env with a fallback. While I never saw $TERMINAL yet, your solution wouldn't hurt and maybe make the default script more readable. But it should not have side effects.